background.js

```diff
@@ -17,6 +33,16 @@ chrome.runtime.onInstalled.addListener((object) => {
         url: chrome.runtime.getURL("html/installed.html")
       });
     }
+
+    for (const opt of popupOptions) {
+      chrome.storage.local.set(opt);
+    }
+  }
+
+  if (object.reason === "update") {
+    for (const opt of popupOptions) {
+      chrome.storage.local.set(opt);
+    }
```

Comment:
We should first check if the options are not already set, so we don't overwrite them when the extension updates

Reply:
Yes, installation don't need, in a first view. Unless browsers save storage from previous uninstalled extensions ahah. (uninstall and install the same extension).

But update, as @ttoino mention, needs some previous check.